app/helpers/device_stories_helper.rb

```diff
@@ -4,13 +4,16 @@ module DeviceStoriesHelper
   def grafana_panel(name)
     panels = {
       cpm: 14,
-      map: 8
+      map: 8,
+      air_quality: 15,
+      air_quality_map: 21
     }
     panels[name]
   end
 
-  def grafana_url(variant = nil)
-    url = ENV['DEVICE_STORIES_GRAFANA_BASE_URL'] || 'https://grafana.safecast.cc/d/DFSxrOLWk/safecast-device-details'
+  def grafana_url(variant = nil, is_airnote = false)
+    url = if is_airnote then ENV['DEVICE_STORIES_GRAFANA_BASE_URL'] || 'https://grafana.safecast.cc/d/7wsttvxGk/safecast-airnote'
```

Comment:
`is_airnote` seems wrong here. Wonder if maybe we should move the `7wsttvxGk/safecast-airnote` vs `DFSxrOLWk/safecast-device-details` distinction into `panels`